Resources/config/extractor_relations.yml

```diff
@@ -1,6 +1,5 @@
 parameters:
     # Sql relations classes
-    ongr_connections.sql_relations.simple_relation.class: ONGR\ConnectionsBundle\Sync\Extractor\Relation\SimpleSqlRelation
-    ongr_connections.sql_relations.composed_relation.class: ONGR\ConnectionsBundle\Sync\Extractor\Relation\ComposedSqlRelation
+    ongr_connections.sql_relations.sql_relation.class: ONGR\ConnectionsBundle\Sync\Extractor\Relation\SqlRelation
```

Comment:
Let's try to break away from sql_relations in general. What is the function of a sql_relations.sql_relation?